src/hooks/useFarmStats.ts

```diff
@@ -58,17 +58,19 @@ type FarmStatsReturnType = {
   totalRewards: BigNumber;
   rewardUnlockRate: BigNumber;
   timeRemaining?: TimeRemaining;
+  totalTime?: number;
 };
 
 export function useFarmStats(farmAddress: string): FarmStatsReturnType {
-  const [apy, setAPY] = React.useState(ZERO);
+  const [apr, setAPR] = React.useState(ZERO);
```

Comment:
Super mega nitpick, but
```suggestion
  const [apr, setApr] = React.useState(ZERO);
```